src/content/viral-amplicon/steps/Step3.md

```diff
@@ -3,33 +3,45 @@ import Link from "$components/Link.svelte";
+  - The output file is a FASTA file
+- `-op position_counts.tsv` tells `viral_consensus` to write the position-by-position nucleotide counts to the file `position_counts.tsv`
+  - This is optional, but the nucleotide counts can be useful for quality-checking the resulting consensus sequence
+- `-oi insertion_counts.json` tells `viral_consensus` to write the insertion counts to the file `insertion_counts.json`
+  - This is optional, but the insertion counts can be useful for quality-checking the resulting consensus sequence
+- `-p $PRIMER_BED` tells `viral_consensus` that our primer BED file is `$PRIMER_BED`
   - A <Link href="https://en.wikipedia.org/wiki/BED_(file_format)">BED file</Link> contains a list of genomic regions
   - The primer BED file contains the the start and end positions (with respect to the reference genome) of our amplicon sequencing primers
-- `-p trimmed.unsorted` tells `ivar` that we want the prefix of our output file (i.e., the file name without the file extension) to be `trimmed.sorted`
-  - `ivar` will automatically add the `.bam` file extension to the prefix we specify
-  - In other words, this will result in an output file named `trimmed.unsorted.bam`
+- `-po 5` tells `viral_consensus` to trim any reads within 5 positions of a primer
+  - In other words, instead of strictly trimming only reads that start within a primer, reads that fall outside of a primer but are close enough (5 positions) will be included in Primer Trimming
+
+After running the above command, we will have successfully called a consensus genome sequence and written the resulting sequence to the file `consensus.fa`.
+
+To see the contents of the consensus genome sequence output file, run the following:
+
+<Execute command="cat consensus.fa" />
+
+This consensus genome FASTA file is the other key result of our amplicon sequencing analysis, and it is what will be used in any downstream analyses (e.g. transmission clustering, phylogenetic inference, lineage assignment, etc.).
+
+We will have also output the nucleotide counts (`position_counts.tsv`) and insertion counts (`insertion_counts.json`), which we can use for quality-checking. Specifically, we can see how many reads supported each nucleotide or insertion in the consensus sequence that we called.
+
+To see the contents of the position counts TSV file, run the following:
 
-After running the above command, we will have successfully trimmed our mapped reads and written the trimmed (no longer sorted) results to the file `trimmed.unsorted.bam`.
+<Execute command="cat position_counts.tsv" />
 
-To see the first few lines of the (unsorted) trimmed BAM output file in the human-readable SAM format, run the following:
+To see the contents of the insertion counts JSON file, run the following:
 
-<Execute command="samtools view -h trimmed.unsorted.bam | \ head -n 5" />
+<Execute command="cat insertion_counts.json" />
```

Comment:
When I run this, I see a JSON where the keys are empty strings. Is that expected, or is that a bug on my end?

```json
  "21447": {
    "": 6
  },
  "21446": {
    "": 6
  },
```

Reply:
Yup, that's expected! That was the critical bug in v0.0.1: any read that *doesn't* have a gap between any two positions should support "no gap between these positions", which I now represent as an "empty-string" insertion. Without this, I was getting a bunch of spurious insertions when we had enough reads (e.g. 10) with the insertion purely by chance, even if we had thousands without the insertion

Reply:
Perfect, thanks!